src/stcal/jump/jump.py

```diff
@@ -338,6 +344,12 @@ def detect_jumps(
             )
             log.info("Total showers= %i", num_showers)
             number_extended_events = num_showers
+
+        # This is where we look for micrometeorite flashes if the triggering
```

Comment:
Since this is done in both conditionals and done last, it should be moved out side the conditionals completely, instead of invoked twice.

The `expand_large_events` and `find_showers` can be moved outside the conditionals for the same reason.

Reply:
Done